javascript/javascript1/week1/dogAge.js

```diff
@@ -0,0 +1,18 @@
+//2. Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2016;
+const dogYearFuture = 2027;
+const shouldShowResultInDogYears = true;
+
+const dogYear;
```

Comment:
This wouldn't work, as you are trying to reassign it. You would have gotten an error if you tried to run your code.

Reply:
changed to let form const in line no. 7. Thank you!!